### PR TITLE
Sort parameters

### PR DESCRIPTION
**Pull request by shimastripe**
## WHY
- Sort parameters at the time of the database query.

## WHAT
- Add `?sort` query.

```
?sort=id,-age,created_at
```

- Minus prefix is descending order.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ Resource name is written in the plural form.
 |---------|-----------|-------|-------|
 |`stream=`|Return JSON in streaming format|`false`|`true`|
 |`q[field_name]=`|A unique query parameter for each field for filtering|(empty)|`q[id]=1,2,5`, `q[admin]=true&q[registered]=true`|
+|`sort=`|Sort parameters|`false`|`id, -age, -created_at`|
```
Comment (dtan4) [inline README.md]: - Default value should be (empty)
- Please explain the correspondences of (none)/`-` prefixes and `asc`/`desc` somewhere. 
- Example shoule be "`id`, `-age`, `id,-created_at`" (imo)

Comment (shimastripe) [inline README.md]: Thx! Please check again. 🙏

File: README.md [hunk 2]
```diff
@@ -206,6 +206,7 @@ Resource name is written in the plural form.
 |---------|-----------|-------|-------|
 |`stream=`|Return JSON in streaming format|`false`|`true`|
 |`q[field_name]=`|A unique query parameter for each field for filtering|(empty)|`q[id]=1,2,5`, `q[admin]=true&q[registered]=true`|
+|`sort=`|Retrieves a list in order of priority. `+` or (none) : ascending. `-` : descending|`empty`|`id`, `-age`, `id,-created_at`|
```
Comment (dtan4) [inline README.md]: \`empty\` -> (empty)

Comment (shimastripe) [inline README.md]: Oh, sorry. I fixed! 🙇

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**